### DISCUSSION TITLE
Error compiling with Azerothcore

### DISCUSSION THREAD

**Discussion by TenebrisWoW** [azerothcore/azerothcore-wotlk#10223] [General]
I try to integrate various new modules into Azerothcore and get the following error message with them:

```
Fehler C2039 "setFaction" ist kein Member von "Creature". modules D:\World of Warcraft Privatserver\Source\modules\mod-premium\src\premium.cpp 319
Fehler C2039 "getFaction" ist kein Member von "Player". modules D:\World of Warcraft Privatserver\Source\modules\mod-premium\src\premium.cpp 319
Fehler C2039 "MonsterWhisper" ist kein Member von "Creature". modules D:\World of Warcraft Privatserver\Source\modules\mod-premium\src\premium.cpp 322
Fehler C2039 "IsSoloLFG" ist kein Member von "lfg::LFGMgr". modules D:\World of Warcraft Privatserver\Source\modules\mod-solo-lfg\src\Lfg_Solo.cpp 38
Fehler C2039 "ToggleSoloLFG" ist kein Member von "lfg::LFGMgr". modules D:\World of Warcraft Privatserver\Source\modules\mod-solo-lfg\src\Lfg_Solo.cpp 40
```

The same with the autobalance_mod!

Can somebody say something about it? They are the last modules that I want to integrate.

This would be the lines from the premium.cpp :

npc->setFaction(player->getFaction());

and

npc->MonsterWhisper(salute, player, false);

This would be the lines from the Lfg_Solo.cpp :

if (sConfigMgr->GetBoolDefault("SoloLFG.Enable", true) != sLFGMgr->IsSoloLFG())

and

sLFGMgr->ToggleSoloLFG();

**Comment by Winfidonarleyan** [comment]
How should we know which version you are using?
And, if module don't compile/work create new issue in module repo
